gym_sts/rl/mvp.py

```diff
@@ -117,8 +117,11 @@ def main(_):
         "mods_dir": os.path.abspath(ENV.value["mods"]),
         "output_dir": output_dir,
     }
+    for key in ["headless", "animate", "reboot_frequency", "reboot_on_error"]:
+        env_config[key] = ENV.value[key]
+
     if SINGLE_COMBAT.value["use"]:
-        env_config["ascension"] = SINGLE_COMBAT.value["ascension"]
+        env_config["ascension"] = ENV.value["ascension"]
```

Comment:
Move this out of the conditional so we can use it even if not testing on single combat.

Reply:
Done